### PR TITLE
Eliminate wrappers

### PR DESCRIPTION
**Pull request by peterbarker**
Also nuke some old comments.

### CODE REVIEW

File: ArduPlane/ArduPlane.cpp [hunk 1]
```diff
@@ -49,8 +49,8 @@ const AP_Scheduler::Task Plane::scheduler_tasks[] = {
     SCHED_TASK(update_alt,             10,    200),
     SCHED_TASK(adjust_altitude_target, 10,    200),
     SCHED_TASK(afs_fs_check,           10,    100),
-    SCHED_TASK(gcs_update,             50,    500),
-    SCHED_TASK(gcs_data_stream_send,   50,    500),
+    SCHED_TASK_CLASS(GCS,                 (GCS*)&plane._gcs,       update,                 50,  500),
+    SCHED_TASK_CLASS(GCS,                 (GCS*)&plane._gcs,       data_stream_send,       50, 1000),
```
Comment (WickedShell) [inline ArduPlane/ArduPlane.cpp]: Execution time estimate changes here for this, and is surprisingly different between the different vehicles. Is that a real difference?

File: ArduPlane/GCS_Mavlink.cpp [hunk 2]
```diff
@@ -1479,36 +1479,11 @@ void Plane::mavlink_delay_cb()
...
-{
-    gcs().update();
-}
-
 /*
   send airspeed calibration data
  */
 void Plane::gcs_send_airspeed_calibration(const Vector3f &vg)
 {
-    gcs().send_airspeed_calibration(vg);
```
Comment (WickedShell) [inline ArduPlane/GCS_Mavlink.cpp]: Pretty sure this was slightly to aggressive a trim?

Comment (peterbarker) [reply ArduPlane/GCS_Mavlink.cpp]: Gah.

I was going to move that call into the library, decided against it... and missed...

File: ArduPlane/ArduPlane.cpp (hunk 1 above)
Comment (peterbarker) [reply ArduPlane/ArduPlane.cpp]: Fixed.

Comment (peterbarker) [reply ArduPlane/ArduPlane.cpp]: .... to be clear, I fixed the incorrect change here.

The delta between the vehicles... well, their data streams are different, I guess...

I wouldn't worry about this too much - when sending is moved to a thread this stuff will go away.

**Comment by peterbarker** [comment]
@magicrub I think the functor-creating-function will complain.... I agree it would be neater to use `gcs()` if possible, but I vaguely recall trying it and failing.

**Pull request merged**